notebooks/00_quick_start/run_notebook_on_azureml.ipynb

```diff
@@ -0,0 +1,290 @@
+{
```

Comment:
<p>we should use tempfile.TemporaryDirectory when creating new directories, this will help with cleanup and potential collisions.</p><p><br></p><p>it would be nice if we could just import a function from generate_conda_file that would give us the conda channels, pip and conda packages, then we can call CondaDepenedencies.create() instead of having to generate a yaml file and modify it.</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/681/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
tempfile.TemporaryDirectory is a great idea. I put that in the notebook.

Now instead of calling generate_conda_file, I used CondaDependencies directly because there's only 1 dependent package to run sar notebook